src/current/v24.3/sso-sql.md

```diff
@@ -38,168 +38,147 @@ The page describes how to configure a cluster for cluster single sign-on using J
-          "kty": "RSA"
-        }
-      ]
-    }
-    ~~~
+    Replace the full contents of `keys` with the list of keys you found previously.
 
     {% include_cached copy-clipboard.html %}
     ~~~sql
     SET CLUSTER SETTING server.jwt_authentication.jwks = '{
       "keys": [
         {
-          "alg": "RS256",
-          "use": "sig",
-          "e": "AQAB",
-          "kty": "RSA",
-          "kid": "7c0b6913fe13820a333399ace426e70535a9a0bf",
-          "n": "lWXY0XOj_ikSIDIvGOhfuRhQJAAj6BWsbbZ6P-PXRclzV32-QLB4GZHPPcH37Lou5pQsTQPvTETAfCLnglIRSbP8x1zA5tUakRlm5RiGF4kcWh5k60x8u0Uslx-d6EueKuY-KLHUVDuMULlHkYAScIdYnXz-Cnr6PFZj8RQezzdPVPH53Q8a_Z9b-vpGzsMS5gszITb-72OQNokojXdPVctl5WzSx-JnWbJxPiwHx_dSWgmTnyiYrZLqrqfampGdroaamtIXy0W8CAe0uCqcD1LunpfX-Q-RD1IycxnEaXSuUKhNhCcxtHWrozEyeD23Zja2WlcvHdYuTzyrvrvS9Q"
-        },
-        {
-          "kid": "6f7254101f56e41cf35c9926de84a2d552b4c6f1",
-          "e": "AQAB",
-          "alg": "RS256",
-          "use": "sig",
-          "n": "oUriU8GqbRw-avcMn95DGW1cpZR1IoM6L7krfrWvLSSCcSX6Ig117o25Yk7QWBiJpaPV0FbP7Y5-DmThZ3SaF0AXW-3BsKPEXfFfeKVc6vBqk3t5mKlNEowjdvNTSzoOXO5UIHwsXaxiJlbMRalaFEUm-2CKgmXl1ss_yGh1OHkfnBiGsfQUndKoHiZuDzBMGw8Sf67am_Ok-4FShK0NuR3-q33aB_3Z7obC71dejSLWFOEcKUVCaw6DGVuLog3x506h1QQ1r0FXKOQxnmqrRgpoHqGSouuG35oZve1vgCU4vLZ6EAgBAbC0KL35I7_0wUDSMpiAvf7iZxzJVbspkQ",
-          "kty": "RSA"
+          ...
         }
       ]
     }';
     ~~~
 
-1. Instead of setting `server.jwt_authentication.jwks` to a list of static signing keys, you can set `server.server.jwt_authentication.jwks_auto_fetch.enabled` to `true` to enable automatic fetching of signing keys for the issuers specified in `server.jwt_authentication.issuers`. Signing keys are fetched from the issuer's`https://{ domain }/.well-known/openid-configuration` endpoint.
+1. <a id="configuration-examples"></a>Instead of setting `server.jwt_authentication.jwks` to a list of static signing keys, you can set `server.server.jwt_authentication.jwks_auto_fetch.enabled` to `true` to enable automatic fetching of signing keys for the issuers specified in `server.jwt_authentication.issuers.configuration`. Signing keys are fetched from the issuer's`https://{ domain }/.well-known/openid-configuration` endpoint.
+    Type                  | Example
+    ----------------------|--------
+    Single issuer URL     | `'https://accounts.google.com/'`
+    Array of issuer URLs  | `'["example.com/adfs","https://accounts.google.com"]'`
+    Map of issuer URLs    | `'{ "issuer_jwks_map": { "https://accounts.google.com/": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" } }'`
```

Comment:
This table is not rendering properly. Sorry, if I missed this on the first review. Maybe you need a space between that and the bullet point?
```suggestion
1. <a id="configuration-examples"></a>Instead of setting `server.jwt_authentication.jwks` to a list of static signing keys, you can set `server.server.jwt_authentication.jwks_auto_fetch.enabled` to `true` to enable automatic fetching of signing keys for the issuers specified in `server.jwt_authentication.issuers.configuration`. Signing keys are fetched from the issuer's`https://{ domain }/.well-known/openid-configuration` endpoint.

    Type                  | Example
    ----------------------|--------
    Single issuer URL     | `'https://accounts.google.com/'`
    Array of issuer URLs  | `'["example.com/adfs","https://accounts.google.com"]'`
    Map of issuer URLs    | `'{ "issuer_jwks_map": { "https://accounts.google.com/": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" } }'`
```

Reply:
Fixed. This is the bane of my existence, what elements need a blank line and which ones break with a blank line.

Reply:
Agree wholeheartedly on what a pain this can be!